common/pom.xml

```diff
@@ -34,6 +34,11 @@
       <version>2.4</version>
     </dependency>
     <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-all</artifactId>
+      <version>4.0.23.Final</version>
+    </dependency>
+    <dependency>
```

Comment:
It was not removed from anywhere else. The reason for this addition was for moving ChannelType to common, which uses netty. However, it was moved from 'servers/', and 'servers/' still needs netty for the dataserver.